peers/validators/canonical_validator_client.go

```diff
@@ -34,7 +34,7 @@ type CanonicalValidatorState interface {
 		ctx context.Context,
 		subnetID ids.ID,
 		pchainHeight uint64,
-	) (avalancheWarp.CanonicalValidatorSet, error)
+	) (validators.WarpSet, error)
```

Comment:
Were the old types left in place in AvalancheGo? I'm surprised this was building at all previously...

Reply:
Left as a type-alias, but might be removed later https://github.com/ava-labs/avalanchego/blob/578d0a915b83cd9ba1243591085ce192d93cdd06/vms/platformvm/warp/validator.go#L33